scripts/local_testnet/README.md

```diff
@@ -5,58 +5,105 @@ This setup can be useful for testing and development.
 
 ## Requirements
 
-The scripts require `lcli`, `lighthouse`, `geth`, `bootnode` to be installed on `PATH`.
+The scripts require `lcli`, `lighthouse`, `geth`, `bootnode` to be installed on `PATH` (run `echo $PATH` to view all `PATH` directories).
 
 
 MacOS users need to install GNU `sed` and GNU `grep`, and add them both to `PATH` as well.
 
-From the
-root of this repository, run:
+The first step is to install Rust and dependencies. Refer to the [Lighthouse Book](https://lighthouse-book.sigmaprime.io/installation-source.html#dependencies) for installation. We will also need [jq](https://jqlang.github.io/jq/), which can be installed with `sudo apt install jq`.
+
+Then, we clone the Lighthouse repository:
+```bash
+cd ~
+https://github.com/sigp/lighthouse.git
+cd lighthouse
+```
```

Comment:
`git clone` is missing here, but I think the above link to "installation from source" covers this step, but I'm not fuss - we could either fix command or just leave it as it was before.

Reply:
thank you, fix in [b70d7c6](https://github.com/sigp/lighthouse/pull/4749/commits/b70d7c642685914d4cc8f7a574ffe375655f3a8c)